src/test/datascience/widgets/commUtils.ts

```diff
@@ -41,26 +41,77 @@ export class Utils {
     public dispose() {
         disposeAllDisposables(this.disposables);
     }
-    public async queryHtml(selector: string, outputId: string) {
+    public async queryHtml(cell: NotebookCell, selector?: string) {
         // Verify the slider widget is created.
         const request = {
             requestId: Date.now().toString(),
-            id: outputId,
```

Comment:
`outputId` is deprecated, and we need a way to hack around this. See notes earlier